skbio/tree/_majority_rule.py

```diff
@@ -124,10 +124,14 @@ def _build_trees(clade_counts, edge_lengths, support_attr):
         Keyed by the frozenset of the clade and valued by the weighted length
     support_attr : str
         The name of the attribute to hold the support value
+    tree_node_class : Python class
```

Comment:
would it be appropriate to describe the type as "TreeNode or subclass of TreeNode"?